extensions/blocks/podcast-player/templates/podcast-header-title.php

```diff
@@ -23,12 +23,22 @@
 ?>
 
 <h2 id="<?php echo esc_attr( $player_id ); ?>__title" class="jetpack-podcast-player__title">
-	<?php if ( ! empty( $track ) && isset( $track['title'] ) ) : ?>
+	<?php if ( isset( $track['title'], $track['link'], $track['src'] ) ) : ?>
```

Comment:
Can I confirm that you want this condition to _fail_ if either `link` OR `src` are not set? Note:

> If multiple parameters are supplied then isset() will return TRUE only if all of the parameters are considered set

For example, is there any occasion where you'd like to display this DOM using `link` if `src` isn't available?

Reply:
Just pushed a change that handles the two independently of each other.